### PR TITLE
chore: use conventional commitlint

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**
chore

**Did you add tests for your changes?**
NA

**If relevant, did you update the documentation?**
No

**Summary**
- Removed unnecessary commit lint packages and adds a single conventional config

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

File: commitlint.config.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 module.exports = {
-    extends: ['cz', '@commitlint/config-lerna-scopes'],
+    extends: ['@commitlint/config-conventional'],
     rules: {
```
Comment (alexander-akait) [inline commitlint.config.js:5]: Let's remove rules, we don't need it and use defaults, also we need fix our doc, `ast` is removed and `tests` renamed to `test`

Comment (anshumanv) [reply commitlint.config.js:5]: yep lets do it

Comment (anshumanv) [reply commitlint.config.js:5]: removing all rules

Comment (anshumanv) [reply commitlint.config.js]: Done

**Review by alexander-akait — APPROVED** [review]
Thanks

**Comment by alexander-akait** [comment]
Thanks

**Pull request merged**